lib/ProductOpener/TaxonomySuggestions.pm

```diff
@@ -97,17 +98,56 @@ Hash of fields that can be taken into account to generate relevant suggestions
 - categories: comma separated list of categories (tags ids or strings in the $search_lc language)
 - shape: packaging shape (tag id or string in the $search_lc language)
 
+=head3 Note
+
+The results of this function are cached using memcached. Restart memcached if you want fresh results
```

Comment:
Shouldn't we also add a time to live to our cached entries ? 1 day seems enough.

So right now as we use it for mongodb we might have old entries served ?

Memcached has built-in support for that in set you can add an expiration time.